tests/core/tools/absoluterectposition.js

```diff
@@ -32,7 +43,7 @@
 
 			assertRect( absoluteRect, expected );
 
-			window.scrollBy( 0, 100 );
+			scrollPositionMock.returns( { x: 0, y: 100 } );
```

Comment:
Since you're explicitly setting the return value of a stub for each test case, I'd favor the entire mock creation to happen in the test case, and not in `setUp` method.

So you assign it like `scrollPositionMock = sinon.stub( window, 'getScrollPosition' ).returns( { x: 0, y: 100 } )`.

Restoring in `tearDown` is perfectly fine, but you'd have to adjust it for case if `scrollPosition` is `null` and also nullify it after it has been restored.

Reason for this is that someone adding a new test case won't be surprised by garbage `undefined` being returned by `getScrollPosition`.